pkg/packagekit/updates.jsx

```diff
@@ -556,13 +565,14 @@ class OsUpdates extends React.Component {
                                 console.warn("GetTransactionList: failed to read PackageKit transaction roles:", ex.message);
                                 // be robust, try to continue with loading updates anyway
                                 this.initialLoadOrRefresh();
+                                this.loadUpdates();
```

Comment:
What dose that do? initialLoadOrRefresh() already calls loadUpdates(). Also, this is only the error handling for the (Should Not Happen™) case that packagekit is very broken. This seems unrelated to this notification work?

Reply:
It's a leftover from some earlier attempts.  Good catch, I'll remove it.